opendm/video/video2dataset.py

```diff
@@ -1,51 +1,18 @@
+import datetime
 from math import floor
 import time
 import cv2
 import os
-
-from .checkers import PercentageBlurChecker, SimilarityChecker, ThresholdBlurChecker
-
-# Input parameters:
-#  input = path to input video file
-#  output = path to output directory
-#  start = start frame index
-#  end = end frame index
-#  output_resolution = Override output resolution (ex. 640x480)
-#  blur_percentage = discard the lowest X percent of frames based on blur score (allowed values from 0.0 to 1.0)
-#  blur_threshold = blur measures that fall below this value will be considered 'blurry' (to be used in exclusion with -bp)
-#  distance_threshold = distance measures that fall below this value will be considered 'similar'
-#  frame_format = frame format (jpg, png, tiff, etc.)
-#  stats_file = Save statistics to csv file
-#  internal_width = We will resize the image to this width before processing
-#  internal_height = We will resize the image to this height before processing
-
-class Parameters:
-
-    def __init__(self):
-
-        self.input = None
-        self.output = None
-        self.start = 0
-        self.end = None
-        self.blur_percentage = None
-        self.blur_threshold = None
-        self.distance_threshold = None
-        self.frame_format = "jpg"
-        self.output_resolution = None
-        self.stats_file = None
-
-        # We will resize the image to this size before processing
-        self.internal_width = 800
-        self.internal_height = 600
-
+from opendm.video.parameters import Parameters
+import pymediainfo
```

Comment:
Can we not include this? It requires libmediainfo, which is going to be a pain to get to work on Windows.

Reply:
I was working and testing on windows, it works pretty well. What alternative do you propose?

Reply:
Ah, never mind; they ship binaries with the wheel. It might be OK then.